OWML.ModHelper.Interaction/InterfaceProxyBuilder.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Text;
+
+namespace OWML.ModHelper.Interaction
+{
+    internal class InterfaceProxyBuilder
```

Comment:
Let's clearly mark this file as based on SMAPI code :)
F.ex. you can link to this page: https://github.com/Pathoschild/SMAPI/tree/c4a82418ac8b09a6965052f5c9173928457fba52/src/SMAPI/Framework/Reflection

Reply:
done!